modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -523,6 +533,66 @@ protected function register_controls() {
 		] );
 
 		$this->end_controls_section();
+
+		$this->start_controls_section( 'section_box_style', [
+			'label' => esc_html__( 'Box', 'elementor' ),
+			'tab' => Controls_Manager::TAB_STYLE,
+		] );
+
+		$this->add_group_control(
+			Group_Control_Background::get_type(),
+			[
+				'name' => 'box_background_color',
+				'types' => [ 'classic', 'gradient' ],
+				'exclude' => [ 'image' ],
+				'selector' => '{{WRAPPER}} .elementor-tabs-content-wrapper',
+			]
+		);
+
+		$this->add_group_control(
```

Comment:
Make sure the labels for Border width and border color are same as in the spec (You can use `'fields_options'` for that)

Reply:
Updated